packages/web/src/scss/components/Stack/README.md

```diff
@@ -31,4 +35,84 @@ Usage with a list:
 </ul>
 ```
 
-👉 Vertical margin of items inside Stack is reset to zero.
+👉 Be aware that the `Stack` component has not default spacing between its children.
+You can use the `Stack--hasSpacing` modifier to add spacing between the children.
+Read below for more details.
+
+## Usage with modifiers classes
+
+Usage with spacing:
+
+```html
+<ul class="Stack Stack--hasSpacing">
```

Comment:
Good point. It should be explicit from class name as well.

Reply:
Discussed personally: it would be confusing rename modifier class to `Stack--hasVerticalSpacing` because component `allows you to compose elements vertically`.